src/schema/provenance_schema.yaml

```diff
@@ -659,6 +659,16 @@ ENTITIES:
           - validate_in_admin_group
         before_property_update_validators:
           - validate_in_admin_group
+      priority_project_list:
+        type: list
+        required: false
+        indexted: false
+        description: The list of priority projects that this Dataset data will be used For
+        mutable: true
```

Comment:
I don't think we have this `mutable` property, only `immutable`.

Reply:
Good catch. I was copying from the details in the card and forgot its immutable not mutable. Fixed this and reversed them (immutable = false rather than mutable = true)